.github/workflows/client-tests.yml

```diff
@@ -53,13 +54,28 @@ jobs:
         sudo ln -s /usr/local/mongodb/mongod /usr/local/bin/mongod
       shell: bash
 
-    - name: Install Vendor Dependencies
+    - name: "Install Mongo Dependencies: windows-latest"
+      if: (matrix.os == 'windows-latest')
+      run: |
+        $url = "https://fastdl.mongodb.org/win32/mongodb-win32-x86_64-2008plus-ssl-3.6.14-signed.msi"
```

Comment:
Could we instead use Chocolatey or scoop to install mongo?

Reply:
Maybe, I'm unsure, worth testing.